examples/kubernetes/connectivity-check/defaults.cue

```diff
@@ -0,0 +1,40 @@
+package connectivity_check
+
+// Default parameters for echo clients (may be overridden).
+deployment: [ID=_]: {
+	// General pod parameters
+	if ID =~ "^pod-to-[-_a-zA-Z0-9]*$" || ID =~ "^host-to-[-_a-zA-Z0-9]*$" {
+		_image:   "docker.io/byrnedo/alpine-curl:0.1.8"
+		_command: ["/bin/ash", "-c", "sleep 1000000000"]
+	}
+
+	// readinessProbe target name
+	if ID =~ "^pod-to-a.*$" || ID =~ "^host-to-a.*$" {
+		_probeTarget: *"echo-a" | string
+	}
+	if ID =~ "^pod-to-b.*$" || ID =~ "^host-to-b.*$" {
+		_probeTarget: *"echo-b" | string
+	}
+
+	// Topology
```

Comment:
AFAIU, the parameters below cannot be overridden, which seems in conflicting with the top-level comment.
Maybe add a comment here to note the exception?

Reply:
Ah, true; they need `| string` to be overridden. Could split this section into two: One with fields that may be overwritten and one with final fields.